climpred/prediction.py

```diff
@@ -2,6 +2,7 @@
 import warnings
 
 import xarray as xr
+import pandas as pd
```

Comment:
Make sure to add `pandas` to the (1) ci/environment yaml reqs, (2) requirements text in the main github directory, (3) requirements text in the docs sub directory.